flagd/pkg/runtime/from_config.go

```diff
@@ -51,13 +51,15 @@ func FromConfig(logger *logger.Logger, version string, config Config) (*Runtime,
 	// register trace provider for the runtime
 	err := telemetry.BuildTraceProvider(context.Background(), logger, svcName, version, telCfg)
```

Comment:
can you check `BuildTraceProvider` internals? I think we can use `IMetricsRecorder` and get rid of `MetricsRecorder` implementation usage.